docs/game_sdk/Networking.md

```diff
@@ -51,90 +51,135 @@ OnLateUpdate()
 
 ## OpenChannel
 
-Opens an unreliable channel to the given session id on the given channel number. Use this type of channel for loss-tolerant data, like player positioning in the world.
+Opens a channel to a user with their given peer on the given channel number.
+
+Unreliable channels—`reliable = false`—should be used for loss-tolerant data, like player positioning in the world. Reliable channels—`reliable = true`—should be used for data that _must_ get to the user, like loot drops!
 
 Returns `void`.
 
 ###### Parameters
 
-| name            | type   | description                     |
-| --------------- | ------ | ------------------------------- |
-| remoteSessionId | UInt64 | the session id to connect to    |
-| channel         | byte   | the channel on which to connect |
+| name      | type   | description                                          |
+| --------- | ------ | ---------------------------------------------------- |
+| peerId    | UInt64 | the peerId of the user to connect to                 |
+| channelId | byte   | the channel on which to connect                      |
+| reliable  | bool   | whether the channel should be unreliable or reliable |
 
 ###### Example
 
 ```cs
 // In reality, you'd fetch this from their lobby metadata
-var playerARemoteSessionId = 1234;
-networkManager.OpenChannel(playerARemoteSessionId, 0);
+var playerAPeerId = 1234;
+networkManager.OpenChannel(playerAPeerId, 0, false);
 ```
 
-## OpenReliableChannel
+## OpenPeer
 
-Opens a reliable channel to the given session id on the given channel number. Use this type of channel for data that _must_ get to the user, like loot drops!
+Opens a network connection to another Discord user.
 
 Returns `void`.
 
 ###### Parameters
 
-| name            | type   | description                     |
-| --------------- | ------ | ------------------------------- |
-| remoteSessionId | UInt64 | the session id to connect to    |
-| channel         | byte   | the channel on which to connect |
+| name   | type   | description                                  |
+| ------ | ------ | -------------------------------------------- |
+| peerId | UInt64 | the peerId of the user to connect to         |
+| route  | byte[] | the route the user is currently broadcasting |
 
 ###### Example
 
 ```cs
-// In reality, you'd fetch this from their lobby metadata
-var playerARemoteSessionId = 1234;
-networkManager.OpenReliableChannel(playerARemoteSessionId, 1);
+networkManager.OpenPeer(1, 2222);
+```
+
+## UpdatePeer
+
+Updates the network connection to another Discord user. This should be called after an `OnRouteChange` event so that your connection to a user is updated to reflect the new route.
```

Comment:
This is a little confusing -- `OnRouteChange` is about *your* route data changing, and you don't need to call `UpdatePeer` when *you* get an `OnRouteChange` event. You should call `UpdatePeer` when you get notified out-of-band (e.g. from a lobby metadata update) that one of the peers you are communicating with publishes a new route.